lib/kino/table.ex

```diff
@@ -3,14 +3,25 @@ defmodule Kino.Table do
 
   @type info :: %{
           name: String.t(),
-          features: list(:refetch | :pagination | :sorting)
+          features: list(:refetch | :pagination | :sorting | :filtering)
         }
 
   @type rows_spec :: %{
           offset: non_neg_integer(),
           limit: pos_integer(),
           order_by: nil | term(),
-          order: :asc | :desc
+          order: :asc | :desc,
+          filter_by: nil | term(),
+          filter_value: nil | term(),
+          filter:
+            :none
```

Comment:
I think we should use `nil` for optional entries, this way if we add add something else to the rows_spec later it's enough to check `spec[:option] == nil` to cover both the none case and the option missing altogether (backwards compatibility).

Same for `order_by` :)